test/k8sT/Policies.go

```diff
@@ -827,78 +736,157 @@ EOF`, k, v)
 		})
 	})
 
-	Context("KubernetesNetworkPolicy between server and client", func() {
+	Context("Namespaces policies", func() {
 
 		var (
-			policy = "allow-ns-b-via-namespace-selector"
+			err         error
+			ciliumPod   string
+			namespace   = "second"
+			appPods     map[string]string
+			appPodsNS   map[string]string
+			clusterIP   string
+			clusterIPNS string
+
+			demoPath         = helpers.ManifestGet("demo.yaml")
+			l3Policy         = helpers.ManifestGet("l3_l4_policy.yaml")
```

Comment:
This variable is called l3Policy, yet the file is l3l4Policy; rename accordingly.